### ISSUE TITLE
[Feature Request] Expose port(s) used by Server class

### ISSUE THREAD

**Issue by C0urante** [moquette-io/moquette#453]
I'd like to be able to instantiate an embedded broker with a port number of 0. This works fine, but then connecting to that embedded broker becomes tricky as I don't know which port it ends up bound to. Something like the following (bare-minimum) sketch would be helpful:

```java
Server mqttBroker = new Server();
Properties serverProperties = new Properties();
serverProperties.put(BrokerConstants.PORT_PROPERTY_NAME, "0");
serverProperties.put(BrokerConstants.SSL_PORT_PROPERTY_NAME, "0");
mqttBroker.startServer(serverProperties);
int tcpPort = mqttBroker.getPort(); // New method for Server class
int sslPort = mqttBroker.getSslPort(); // New method for Server class
```

I'd try to tackle this myself with a PR but I'm not terribly familiar with Netty and haven't been able to find a satisfactory way to implement this functionality yet.

**Comment by andsel** [comment]
Thanks good suggestion!!
 Andrea

**Comment by andsel** [comment]
Fixed with 8bd0dc785290a34df0c9b3a8bf42a76d364561c2
